internal/buildscripts/packaging/installer/install.sh

```diff
@@ -333,8 +335,8 @@ ensure_not_installed() {
       echo "Please uninstall auto instrumentation, or try running this script with the '--uninstall' option." >&2
       exit 1
     fi
-    if command -v npm >/dev/null 2>&1 && npm ls --global @splunk/otel >/dev/null 2>&1; then
-      echo "The @splunk/otel npm package is already globally installed." >&2
+    if [ -n "$npm_path" ] && $npm_path ls --prefix $node_install_prefix @splunk/otel >/dev/null 2>&1; then
```

Comment:
not an issue functionally but I think `--prefix` is discouraged: https://github.com/npm/cli/issues/1368#issuecomment-1241076700

Reply:
Let me change it to use `cd` just to be safe in case they ever decide to deprecate the `--prefix` option.